salt/modules/jira_mod.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8 -*-
+'''
+JIRA Execution module
+=====================
+
+.. versionadded:: Fluorine
+
+Execution module to manipulate JIRA tickets via Salt.
+
+This module requires the ``jira`` Python library to be installed.
+
+Configuration example:
+
+.. code-block:: yaml
+
+  jira:
+    server: https://jira.atlassian.org
+    username: salt
+    password: pass
+'''
+from __future__ import absolute_import
```

Comment:
`print_function` and `unicode_literals` need to be added here for imports

Reply:
Fixed @rallytime